examples/visualization/styles/helpers/color-bins-style.html

```diff
@@ -45,18 +45,7 @@
         }
       });
 
-      const availabilityStyle = carto.viz.colorBinsStyle('availability_365', {
-        // breaks: [0, 100, 200, 300, 365],
-        // Methods allowed are quantiles, equal and stdev
-        method: 'quantiles',
-        
-        opacity: 0.5,
-        // palette accepts an array of HEX color strings, or a string pointing to a CARTOColors palette
-        palette: 'teal',
-        // Color applied to features which the attribute value is null.
-        nullColor: '#ccc',
-      });
-
+      const availabilityStyle = carto.viz.colorBinsStyle('availability_365');
```

Comment:
It's nice to see the simplest example.
 
But considering that lots of users will copy & paste, I think we should keep the more verbose one somehow. If not, the user will have to go to the reference to see those extra params.

I tried to do that, in the map helper example, doing this:

```js
      const deckMap = carto.viz.createMap();
      // const deckMap = carto.viz.createMap({ basemap: 'positron'});
      // const deckMap = carto.viz.createMap({ basemap: 'darkmatter', view: { zoom: 4 } });
      // const deckMap = carto.viz.createMap({ basemap: 'positron', view: { zoom: 4, longitude: 3, latitude: 40, pitch: 45, bearing: 30 }, container: 'map' });
```

Reply:
I think the opposite :) Since it is a helper example, I will try to show how simple the helper is and the result. In case we want to add more, I would do it in an `advance use examples`

Reply:
hahaha, that's funny I did it at https://github.com/CartoDB/toolkit/pull/102/commits/74521fd7538cb4e54c45991b01450c5781530ba7 and I thought it was a little bit overwhealming

Reply:
Applying what "Simón says" (clap your hands... 👏 ) could be useful, having 2 examples per helper, one with all defaults, and one / two with more params. Maybe we can ping other people for opinion on this tomorrow

Reply:
Make sense to me the idea of advance examples, but let do that in a separated issue.